fe1-web/parts/lao/socialMedia/SocialProfile.tsx

```diff
@@ -1,26 +1,58 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import { PublicKey } from 'model/objects';
+import { Chirp, ChirpState } from 'model/objects/Chirp';
 import TextBlock from 'components/TextBlock';
-import { StyleSheet, View, ViewStyle } from 'react-native';
+import ChirpCard from 'components/ChirpCard';
+import ProfileIcon from 'components/ProfileIcon';
+import {
+  FlatList, ListRenderItemInfo, Text, View,
+} from 'react-native';
 import STRINGS from 'res/strings';
-import SocialUserProfile from './SocialUserProfile';
-
-const styles = StyleSheet.create({
-  textUnavailableView: {
-    alignSelf: 'center',
-    width: 600,
-    marginTop: 20,
-  } as ViewStyle,
-});
+import { makeChirpsListOfUser } from 'store';
+import { useSelector } from 'react-redux';
+import socialMediaProfile from 'styles/stylesheets/socialMediaProfile';
 
 /**
  * UI for the profile of the current user.
  */
+
+const styles = socialMediaProfile;
+
 const SocialProfile = (props: IPropTypes) => {
   const { currentUserPublicKey } = props;
-  if (currentUserPublicKey.valueOf() !== '') {
-    return <SocialUserProfile userPublicKey={currentUserPublicKey} />;
+  if (currentUserPublicKey && currentUserPublicKey.valueOf() !== '') {
```

Comment:
you might be able to save a bit of indentation by turning this into the opposite condition and returning the `social_media_your_profile_unavailable` view here.